docs/stencil-docs/developing-further/customizing-emails.md

```diff
@@ -1,18 +1,29 @@
-# BigCommerce Email Templates
+# BigCommerce Legacy Email Templates
 
 <div class="otp" id="no-index">
 
```

Comment:
Should we add a callout for where to find documentation for the newer emails?

Reply:
We do call this out first thing in the article. It will be wrapped in a callout.